components/compositing/compositor.rs

```diff
@@ -1641,6 +1701,16 @@ impl IOCompositor {
         };
         webview_renderer.notify_input_event_handled(input_event_id, result);
     }
+
+    /// Disable LCP feature when the user interacts with the page.
+    fn disable_lcp_calculation(&mut self) {
+        let mut current_preferences = servo_config::prefs::get().clone();
```

Comment:
This looks wrong to me. We should stop LCP collection just for a specific `WebViewId`, but this function disables it globally and permanently.